tools_hive/conf/SiteDefs.pm

```diff
@@ -126,6 +127,18 @@ sub update_conf {
   $SiteDefs::ENSEMBL_LD_LSF_TIMEOUT            = undef;                                            # Max timelimit a LD job is allowed to run on LSF
   $SiteDefs::ENSEMBL_LD_ANALYSIS_CAPACITY      = 500;                                              # Number of jobs that can be run parallel in the LD queue (LSF or LOCAL)
 
+  # Variant Recoder configs
+  $SiteDefs::ENSEMBL_VR_RUN_LOCAL              = 1;
+  $SiteDefs::ENSEMBL_VR_QUEUE                  = 'highpri';
+  $SiteDefs::ENSEMBL_VR_LSF_TIMEOUT            = undef;
+  $SiteDefs::ENSEMBL_VR_ANALYSIS_CAPACITY      = 500;
+  $SiteDefs::ENSEMBL_VR_SCRIPT_DEFAULT_OPTIONS = {
```

Comment:
if these are undef we probably need to remove it completely but not sure if this has been added intentionally.